exercises/triangle/src/example.c

```diff
@@ -0,0 +1,23 @@
+#include "triangle.h"
+
+int triangle_equality(double a, double b, double c)
```

Comment:
This function should be declared as `static int triangle_equality`. We're trying to be more consistent with that :)